app/lib/common/widgets/html_editor/components/mention_block.dart

```diff
@@ -0,0 +1,92 @@
+import 'package:acter/common/providers/room_providers.dart';
+import 'package:acter/common/widgets/html_editor/models/mention_block_keys.dart';
+import 'package:acter_avatar/acter_avatar.dart';
+import 'package:appflowy_editor/appflowy_editor.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+
+class MentionBlock extends ConsumerWidget {
```

Comment:
The UI component of the inline mentions in editor (of how it should be displayed when  triggered)